front/app/containers/Admin/sideBar/MenuItem.tsx

```diff
@@ -96,27 +104,58 @@ const MenuItem = ({ navItem }: Props) => {
     item: { type: 'route', path: navItem.link },
   });
 
+  // Temporary proposal warning implementation, will be removed together with the navbar item
+  // after users have had enough time to get used to the feature
+
+  const isProjectProposalsEnabled = useFeatureFlag({
+    name: 'proposals_participation_method',
+  });
+
+  const isItemDisabled =
```

Comment:
I realize this is not the cleanest implementation but it will soon be deleted so I opted for the fastest solution here.